lib/types.d.ts

```diff
@@ -0,0 +1,91 @@
+import builder = require('xmlbuilder');
+export declare const enum EnumChangefreq {
+    DAILY = "daily",
+    MONTHLY = "monthly",
+    ALWAYS = "always",
+    HOURLY = "hourly",
+    WEEKLY = "weekly",
+    YEARLY = "yearly",
+    NEVER = "never"
+}
+export declare const CHANGEFREQ: EnumChangefreq[];
+export declare const enum EnumYesNo {
+    YES = "yes",
+    NO = "no"
+}
+export declare const enum EnumAllowDeny {
+    ALLOW = "allow",
+    DENY = "deny"
+}
+export declare type ICallback<E extends Error, T> = (err: E, data?: T) => void;
```

Comment:
function will not always be called with E if working properly. As @RyuuGan pointed out in another PR. Not sure if that means it should be `E|null` or just marked as optional

Reply:
don't worry ICallback allow input null, this is make sure err is insof Error Class

Reply:
👍